pkgs/native_assets_cli/lib/src/locking/locking.dart

```diff
@@ -73,6 +80,8 @@ Future<T> _runUnderFileLock<T>(
         );
         printed = true;
       }
+      // Don't busy wait, give the CPU some rest.
+      // Magic constant taken from flutter_tools for startup lock.
       await Future<void>.delayed(const Duration(milliseconds: 50));
```

Comment:
(*Someone* should make an extension on `Duration` to wait for it, like `Future<void> get delay => Future<void>.delayed(this, () {});`. :grin:)